.github/workflows/rust.yml

```diff
@@ -137,6 +137,17 @@ jobs:
         run: |
           forest-cli sync wait
           forest-cli --chain calibnet db stats
+      - name: export snapshot and sync forest with exported snapshot
+        run: |
+          rm /tmp/snapshots/*
+          forest-cli snapshot export -o /tmp/snapshots
+          forest --chain calibnet --target-peer-count 50 --encrypt-keystore false --halt-after-import --import-snapshot /tmp/snapshots/*.car
```

Comment:
I wonder if we can reuse steps within a job, by referring to their ids. Is this possible?

Reply:
You can definitely re-use the database that was created in the previous steps. No need to import another snapshot.